zookeeper-server/src/main/java/org/apache/zookeeper/server/watch/WatcherCleaner.java

```diff
@@ -161,10 +163,10 @@ public void doWork() throws Exception {
                         long startTime = Time.currentElapsedTime();
                         listener.processDeadWatchers(snapshot);
                         long latency = Time.currentElapsedTime() - startTime;
-                        LOG.info("Takes {} to process {} watches", latency, total);
+                        LOG.info("Takes {} to process {} watchers", latency, total);
```

Comment:
Watches seems to be more reasonable here, watcher maps to a single client session, and it can watch multiple paths, so have multiple watches on a single watcher, the total value is the total watches count not watcher.